### PR TITLE
Hfeyp 200 read more

### PR DESCRIPTION
**Pull request by guy-roberts**
### Context
Adds a way for editors to add a "Read More" heading with a details element that is hidden

### Changes proposed in this pull request
The GovSpeak markdown now accepts the summary and details tags

### Guidance to review
Check the Read More is styled as in the prototype

Screenshot 2021-03-03 at 12 45 34

Check the Read More link opens and closes the details
Check that the details are formatted correctly

The markdown for a summary/details section has to start the line with no spaces !
`<details class="govuk-details" data-module="govuk-details">
<summary class="govuk-details__summary">
<span class="govuk-details__summary-text">
                 Read more
</span>
</summary>
<div class="govuk-details__text">
<p>Children’s back-and-forth interactions from an early age form the foundations for language and cognitive development.</p>
<p>The number and quality of the conversations they have with adults and peers throughout the day in a language-rich environment is crucial.</p>
</div>
</details>`

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -25,6 +25,8 @@ class Application < Rails::Application
     config.load_defaults 6.0
     config.autoloader = :classic
 
+    config.action_view.sanitized_allowed_tags = ['details', 'summary', 'a']
```
Comment (tonyto) [inline config/application.rb:28]: Why is this required?

Comment (guy-roberts) [reply config/application.rb:28]: Oh, I think its not needed, I will check and remove it.

Thanks for spotting that

(Left over from attempts to fix the issue)

**Review by tonyto — APPROVED** [review]
